### PR TITLE
Update to-integrations-repo script to handle logstash docs

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

**Comment by asuresh4** [comment]
@keitwb - I'll be adding subsequent commits to handle config tables. But it would great if you could take a look. Results of running the script: https://github.com/signalfx/integrations/pull/759

File: scripts/docs/templates/integrations_doc.txt [hunk 1]
```diff
@@ -0,0 +1,157 @@
+
```
Comment (keitwb) [inline scripts/docs/templates/integrations_doc.txt]: Call this `integrations_docs.md.jinja2` to make it clear it is a template for a markdown file.

File: scripts/docs/to-integrations-repo_v2 [hunk 2]
```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```
Comment (keitwb) [inline scripts/docs/to-integrations-repo_v2]: Just replace the old script whenever you are done with these changes.

File: scripts/docs/to-integrations-repo_v2 [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+# Syncs all the relevant docs/metadata from the agent to the integrations repo.
+
+import copy
+import json
+import os
+import re
+import sys
+
+from collections import OrderedDict
+from jinja2 import Environment, FileSystemLoader
```
Comment (keitwb) [inline scripts/docs/to-integrations-repo_v2]: Add the jinja dep to a requirements.txt file somewhere and pin it to a specific version.

File: scripts/docs/templates/integrations_doc.txt (hunk 1 above)
Comment (asuresh4) [reply scripts/docs/templates/integrations_doc.txt]: updated

File: scripts/docs/to-integrations-repo_v2 (hunk 2 above)
Comment (asuresh4) [reply scripts/docs/to-integrations-repo_v2]: yep, that's the plan. wanted to make it easier to review.

File: scripts/docs/to-integrations-repo_v2 (hunk 3 above)
Comment (asuresh4) [reply scripts/docs/to-integrations-repo_v2]: done

File: scripts/docs/to-integrations-repo_v2 [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+    j2_env.filters["surround_by_tick"] = surround_by_tick
+    j2_env.filters["has_atleast_one_monitor_with_section"] = has_atleast_one_monitor_with_section
+    j2_env.filters["send_all_metrics_from_all_monitors"] = send_all_metrics_from_all_monitors
+
+    return j2_env
+
+
+def get_template():
+    j2_env = get_jinja_env()
+    template = j2_env.get_template("integrations_doc.md.jinja2")
```
Comment (keitwb) [inline scripts/docs/to-integrations-repo_v2]: Make sure this script will work the same no matter what the PWD is when run.

Comment (asuresh4) [reply scripts/docs/to-integrations-repo_v2]: A file system loader object is passed to the environment `j2_env`. Any template in the path passed into the filesystem loader object, can be accessed in this manner. `INTEGRATIONS_DOC_TEMPLATE_DIR` is passed in and is derived from the current script directory. That should suffice right?

Comment (keitwb) [reply scripts/docs/to-integrations-repo_v2]: I suppose so, just make sure that you can run the script equally well when your shell is in the root of the repo as from the same directory where the script is.

File: docs/monitors/logstash-tcp.md [hunk 5]
```diff
@@ -93,6 +93,9 @@ output {
   }
 }
 ```
+Once Logstash is configured with the above configuration. The logstash-tcp monitor
+will collect `logins.count` and `process_time.<timer_field>`. See config options
+for default
```
Comment (keitwb) [inline docs/monitors/logstash-tcp.md]: Period needed at end of sentence.

Comment (keitwb) [reply docs/monitors/logstash-tcp.md]:
```suggestion
for default values.
```

Comment (asuresh4) [reply docs/monitors/logstash-tcp.md]: Done

**Pull request merged**